app/utilities/schema.py

```diff
@@ -122,14 +123,10 @@ def load_schema_from_metadata(metadata):
         return schema
 
     return load_schema_from_name(
-        metadata.get("schema_name"), language_code=metadata.get("language_code")
+        metadata.get("schema_name"), language_code=language_code
     )
```

Comment:
Same as above. Language should come from metadata if not passed in.

Reply:
Updated the method to get from metadata if the language code is not passed 👍